service/src/main/java/bio/terra/cbas/dependencies/leonardo/AppUtils.java

```diff
@@ -169,11 +173,18 @@ public String findUrlForWds(List<ListAppResponse> apps) throws DependencyNotAvai
   public String findUrlForCromwell(List<ListAppResponse> apps)
       throws DependencyNotAvailableException {
     ListAppResponse foundApp = findBestAppForAppType(apps, AppType.CROMWELL);
+    Object proxyUrl;
 
+    if (Objects.requireNonNull(foundApp.getProxyUrls()).containsKey("cromwell-writer")) {
+      proxyUrl = "cromwell-writer";
+    } else {
+      proxyUrl = "cromwell";
+    }
```

Comment:
TOL: I am wondering if it would be better to figure out the proxy url key based on the app type of `foundApp` like this? Because it is possible that the `foundApp` doesn't have a `cromwell` or `cromwell-runner` key yet because it was not deployed properly 🤔 
```
If (foundApp.appType == CROMWELL_RUNNER_APP) {
  proxyUrlKey = "cromwell-runner"
}
else {
  proxyUrlKey = "cromwell"
}
```